src/main/java/server/main.java

```
@@ -27,6 +27,7 @@ public static void main(String[] args) {
                 try {
                     Thread.sleep(1000);
                 } catch (InterruptedException te) {
+                    // Catch block is currently empty
```

We should probably explain why this doesn't need to be handled, basically from what I understand it'll only be interrupted if the program is stopped while it's sleeping, in which case the program is already exiting and we don't need to report any errors.